.github/workflows/daily-build.yml

```diff
@@ -0,0 +1,64 @@
+    - cron: '30 2 * * *'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+    # This workflow contains a single job called "build"
+    build:
+        # The type of runner that the job will run on
+        runs-on: ubuntu-latest
+
+        # Steps represent a sequence of tasks that will be executed as part of the job
+        steps:
+            # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+            - uses: actions/checkout@v2
+
+            # Set up Java Environment
+            - name: Set up JDK 11
+              uses: actions/setup-java@v1
+              with:
+                java-version: 11
+            
+            # Grant execute permission to the gradlew script
+            - name: Grant execute permission for gradlew
+              run: chmod +x gradlew
+
+            # Build the project with Gradle
+            - name: Build with Gradle
+              env:
+                packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
+                packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
+                JAVA_OPTS: -DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true
+              run: |
+                ./gradlew build
+            # Build the ballerina project
+            - name: Ballerina Build
+              uses: ballerina-platform/ballerina-action/@nightly
+              with:
+                  args:
+                      build -c ./peoplehr
+              env:
+                JAVA_HOME: /usr/lib/jvm/default-jvm
+                BASE_URL: ${{ secrets.BASE_URL }}
+                API_KEY: ${{ secrets.API_KEY }}}
+                EMAIL_ADDRESS: ${{ secrets.emailAddress }}
+                PASSWORD: ${{ secrets.PASSWORD }}
+                QUERY_NAME: ${{ secrets.QUERY_NAME }}
+
+            # Send notification when build fails
+            - name: Notify failure
+              if: ${{ failure() }}
+              run: |
+                curl \
+                -X POST 'https://chat.googleapis.com/v1/spaces/${{secrets.BALLERINA_CHAT_ID}}/messages?key=${{secrets.BALLERINA_CHAT_KEY}}&token=${{secrets.BALLERINA_CHAT_TOKEN}}' \
+                --header 'Content-Type: application/json' \
+                -d '{"text": "*module-ballerinax-peoplehr* daily build failure \nPlease visit <https://github.com/ballerina-platform/module-ballerinax-peoplehr/actions?query=workflow%3A%22Daily+build%22|the daily build page> for more information"}'
+
+                curl \
+                -X POST 'https://chat.googleapis.com/v1/spaces/${{secrets.CONNECTOR_CHAT_ID}}/messages?key=${{secrets.CONNECTOR_CHAT_KEY}}&token=${{secrets.CONNECTOR_CHAT_TOKEN}}' \
+                --header 'Content-Type: application/json' \
+                -d '{"text": "*module-ballerinax-peoplehr* daily build failure \nPlease visit <https://github.com/ballerina-platform/module-ballerinax-peoplehr/actions?query=workflow%3A%22Daily+build%22|the daily build page> for more information"}'
```

Comment:
Now we dont send notification to 2 groups.
We send notification only to one common group.
Please check the implementation in other repos.